crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -349,6 +371,18 @@ impl RealRibosome {
         Ok((instance, context_key))
     }
 
+    fn cranelift_fn() -> fn() -> Cranelift {
+        || {
+            let cost_function = |_operator: &Operator| -> u64 { 1 };
+            // @todo 10 giga-ops is totally arbitrary cutoff so we probably
+            // want to make the limit configurable somehow.
+            let metering = Arc::new(Metering::new(10_000_000_000, cost_function));
```

Comment:
Should we set this default value as a constant somewhere that has the possibility for being overridden with config settings rather than hard coded inline?

Reply:
+1 on making this overridable at runtime so that people who test this function have a way to iteratively figure out the amount of ops their happ scenario consume.

Reply:
yes of course but i was keeping the scope down and confirmed in chat that i was hardcoding it short term

as this happens on the host side we can ship some config later without disrupting happs